commit da78498cb3f18fe1391de9387f88629f56e4aa8c
Author: Rafael Roquetto <rafael.roquetto.qnx@kdab.com>
Date:   Mon Dec 10 18:15:39 2012 -0200

    Blackberry: remove JSON plugin reference from .pro
    
    Remove a reference to blackberry.json from the Blackberry bearer engine
    plugin. This reference only makes sense under Qt5
    
    Change-Id: I1592a17e38cb80b3fb138fc0538f3b4ea33739c8
    Reviewed-by: Sean Harmer <sean.harmer@kdab.com>

diff --git a/src/plugins/bearer/blackberry/blackberry.pro b/src/plugins/bearer/blackberry/blackberry.pro
index 4b24caa..e8fad69 100644
--- a/src/plugins/bearer/blackberry/blackberry.pro
+++ b/src/plugins/bearer/blackberry/blackberry.pro
@@ -14,8 +14,6 @@ SOURCES += qbbengine.cpp \
            ../qnetworksession_impl.cpp \
            main.cpp
 
-OTHER_FILES += blackberry.json
-
 QTDIR_build:DESTDIR = $$QT_BUILD_TREE/plugins/bearer
 target.path += $$[QT_INSTALL_PLUGINS]/bearer
 INSTALLS += target
